### ISSUE TITLE
Particles don't remove in /portal remove command

### ISSUE THREAD

**Issue by rodel77** [trentech/ProjectPortals#37]
Hi, i am testing portals with particles, i wan't to see all particles and decide one, when i change the particle the old still there, then i try to remove the portal, particles still there

You can fix it pls? And if it's posible add MySQL support, i think some people go to use it!

**Comment by NemoN** [comment]
same problem here, please fix +1

**Comment by lantr** [comment]
Same issue here.   If you restart the server and log back in you will see the new particle effect if you changed it

**Comment by trentech** [comment]
I've already fixed this in upcoming update. I only get weekends to work on this stuff so.. Saturday hopefully

**Comment by EmanYm** [comment]
This is still an issue in PJP 0.13.6
>/portal particle play ENCHANTING_GLYPHS RED
Get white enchanting glyphs
/portal particle play NONE
nothing happened
/portal particle play PORTAL2
get grey enchanting glyphs in addition to the white ones
/portal particles play NONE
nothing happens
restart client, no change.

The only thing I can do to remove the particles is to remove the portal and re-create it.
And I still have that stupid "purple ketchup" behind them

**Comment by EmanYm** [comment]
I left the server over night, and it seems that waiting for the server to do it's nightly restart fixed the error with the ketchup.
I have changed the partical a couple times.  The purple comes back and I have to restart the server to remove it again.
I noticed the bubbles are animated way to quick (like it's missing most of its frames).  Not sure if that's your code or Minecraft code, but it looks like a bad .gif from the mid 90's
